doc/changes.rst

```diff
@@ -203,6 +203,9 @@ Changes to individual commands
 
 ``history``
   * ``undo`` subcommand now accepts ``--ignore-extras`` and ``--ignore-installed`` like original ``history replay`` command.
+  * ``store`` subcommand now outputs a directory with transaction JSON file instead of a single transaction JSON file directly.
+  * ``replay`` subcommand was moved to a standalone ``replay`` command, that now accepts a path to a directory instead of a single file.
```

Comment:
Minor or optional - What about to replace `of a single file` by `of a file path`.